src/components/footer/style.module.css

```diff
@@ -12,7 +12,25 @@
 		margin-left: 0.625rem;
 	}
 
+	/**
+	 * Ideally kept sync'd with the markdown links:
+	 * https://github.com/preactjs/preact-www/blob/f68e65aadbd19b042921cc175aa256f3aa248768/src/style/markdown.css#L18
+   */
```

Comment:
As this comment would suggest, our new footer link styles should match the markdown links but we're relying on copy/paste for the moment. It's only a few lines and I don't imagine it'll change often.